scripts/create-component/create-component.ts

```diff
@@ -264,7 +264,6 @@ const projectsWithStartCommand = Object.entries(allPackages)
     ([pkg, info]) =>
       !ignoreProjects.includes(pkg) &&
       info.packagePath.startsWith('packages') &&
-      info.packageJson.dependencies &&
-      info.packageJson.dependencies['@fluentui/react-compose'] !== undefined,
+      info.packageJson.dependencies?.['@fluentui/react-compose'] !== undefined,
```

Comment:
I don't understand the whole context of this check, but we should not depend here on `@fluentui/react-compose`: for converged components we should use `@fluentui/react-utils` 😺

Reply:
https://github.com/microsoft/fluentui/pull/16996#pullrequestreview-590590737